terraform/main.tf

```diff
@@ -39,7 +46,7 @@ module "development" {
   port = 2001
 
   listener_port = 80
-  path_pattern  = "thc"
+  path_pattern  = "/thc*"
```

Comment:
What's the impact of this change?

Reply:
New value matches the current setting in AWS. It was updated manually at some point so the source is out of date.

Reply:
Yikes. Ok. Thanks.